R/pkg/R/SQLContext.R

```diff
@@ -835,7 +843,7 @@ loadDF <- function(x, ...) {
 #' @note createExternalTable since 1.4.0
 createExternalTable.default <- function(tableName, path = NULL, source = NULL, ...) {
   sparkSession <- getSparkSession()
-  options <- varargsToEnv(...)
```

Comment:
Yeap, it seems there is one case left[1].

[1]https://github.com/apache/spark/blob/d2fde6b72c4aede2e7edb4a7e6653fb1e7b19924/R/pkg/R/group.R#L104